core/libelf/build

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-sed -i 's/-I/-isystem /g' libelf.pc.in
+printf ",s/-I/-isystem \nw\nq" | ed libelf.pc.in >/dev/null
```

Comment:
This doesn't work as it doesn't replace all occurrences of `-I`. This affects all cases of `sed ..../g`.